docs/code_of_conduct.md

```diff
@@ -20,13 +20,22 @@ Private or public harassment of any kind will not be tolerated. Since harassment
 
 Additionally, spam and other content which disrupts or prevents LDK contributors from working on LDK is not acceptable.
 
+In case of a concrete issue that raises a deadlock about the application of the code of conduct, 2 additional members should be selected by consensus of the code of conduct team from the community of
+active LDK contributors to join the team in its decision-making process to adjudicate the case. The decision of the majority of the 5 members is binding.
+
 ## The Code of Conduct Team
 
 A small team of LDK contributors has volunteered to enforce the LDK Code of Conduct. If you feel like a community member has engaged in inappropriate behavior, please don't hesitate to contact one of the following LDK contributors via email or on Discord:
  * Matt Corallo - ldkcocpoc on mattcorallo.com
  * Val Wallace - vwallace on protonmail.com
  * Devrandom - devrandom99 on proton.me
 
+The Code of Conduct team should be renew every 3 years with a transparent nomination process opened to the whole community of active LDK contributors.
```

Comment:
nit:

```suggestion
The Code of Conduct team should be renewed every 3 years with a transparent nomination process open to the whole community of active LDK contributors.
```

Reply:
I like the staggered approach mentioned, so happy to have that included here :)

As for nominations, I do hear the concerns of Matt around people really vying for the position (who actually might have bad intents). I don't know of a good solution to this, but for me at least the staggered approach and having it 3 years instead of 2 can provide some stability and handover if other candidates ever wish to be nominated.

I think we need to address what happens in the special case of a current CoC member who wants to step down, is there an early nomination? Are there acting-members on the bench to fill that role until the end of a term?

Reply:
Included the staggered approach mention.

On the concern raised by Matt about people vying for the position, I think requesting updated technical proof-of-work both for CoC members and for the active LDK contributors taking part to the nomination process ensure those nominated to the roles are productive contributors giving priority to technical development over playing bureaucracy games.

In case of current CoC member who wants to step down, added a new sentence where the role can be fulfilled with a new nomination instantly. I think it’s better to guarantee a minimal number of people part of the Code of Conduct team as you can expect better decisions to be taken from the rational discussion of multiple independent viewpoint.

Overall a code of conduct is an attempt to find a reasonable middle ground among the the disagreements regarding expression in a middle-scale and diverse community of people from all around the world, IMHO.